spec/spec.md

```diff
@@ -215,7 +225,7 @@ ISO and IEC maintain terminological databases for use in standardization at the
 
 [[def: Live-Attack]]
 
-~ an attack that compromises either the current signing keys used to sign non-establishment events or he current pre-rotated keys needed to sign a subsequent establishment event. See (Security Properties of Prerotation)[#live-attacks].
+~ an attack that compromises either the current signing keys used to sign non-establishment events or he current pre-rotated keys needed to sign a subsequent establishment event. See [Security Properties of Prerotation](#live-attacks).
```

Comment:
... sign non-establishment events or the current pre-rotated...

You are missing the `t` in the.